FirebaseRemoteConfig/Tests/SwiftAPI/RemoteConfigConsole.swift

```diff
@@ -0,0 +1,242 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+class RemoteConfigConsole {
+  private let projectID: String
+  private var latestConfig: [String: Any]!
+
+  public var requestTimeout: TimeInterval = 10
+
+  private var consoleURL: URL {
+    let api = "https://firebaseremoteconfig.googleapis.com"
+    let endpoint = "/v1/projects/\(projectID)/remoteConfig"
+    return URL(string: api + endpoint)!
+  }
+
+  private lazy var accessToken: String = {
+    guard let fileURL = Bundle(for: type(of: self))
+      .url(forResource: "AccessToken", withExtension: "json") else {
```

Comment:
I've since reran it and it indented it like that. I agree though, an extra two spaces inside the guard's returning clause helps make things look clearer.